src/help/getRenderProperties.js

```diff
@@ -52,12 +52,18 @@ function getRenderProperties(element){
 		};
 	}
 	// If canvas (in node)
-	else if(element.getContext){
+	else if(element && element.getContext){
 		return {
 			element: element,
 			renderer: renderers.CanvasRenderer
 		};
 	}
+	else if(element && typeof element === 'object') {
```

Comment:
So would it be sufficient to say that object renders have to be an empty object? I can imagine situations where somebody might want to add it onto an existing object. As it stands that would be work fine. That'd probably be a bad idea in most situations though.